10_code/60_WA_and_controls_pop_merge.py

```diff
@@ -0,0 +1,79 @@
+import pandas as pd
+import numpy as np
+
+
+##import data
+pop = pd.read_csv(
+    "https://raw.githubusercontent.com/MIDS-at-Duke/pds-2022-red-team/main/20_intermediate_files/pop_merged.csv"
+)
+WA_pres = pd.read_csv(
+    "https://raw.githubusercontent.com/MIDS-at-Duke/pds-2022-red-team/wa_and_comparisons/20_intermediate_files/Opioid_Prescriptions_WA_comparisons.csv"
+)
+
+
+WA_col = ["WA", "OR", "CA", "NV", "ID", "MT"]
+
+# select only the states in Florida_col in population
+WA_pop = pop[pop["State"].isin(WA_col)]
+
+
+##find all county names ending with 'County' or 'Parish' except for Carson City
+WA_pop[
+    (~WA_pop["County"].str.contains("County"))
```

Comment:
It will likely involve a lot more data cleanup work but one thing to consider here is to use FIPS code rather than aligning county names between population and shipment datasets.